pom.xml

```diff
@@ -754,6 +754,14 @@
                                             </fileset>
                                         </replaceregexp>
 
+                                        <replaceregexp byline="true" encoding="UTF-8">
```

Comment:
This script should ADD new changelog entry — date-time, author, version, changes and so on.

Reply:
Added scripts that will add changelog entry below